index.html

```diff
@@ -5,14 +5,135 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
     <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
-<body>
-
+<body class="container">
+    <header class="row">
+        <div class="col-3">
+            <p class="name">Alice Rose</p>
+            <p class="job">Personal Life Coach</p>
+        </div>
+        <nav class="col-6">
+            <ul class="navlist">
+                <li>Home</li>
+                <li>About</li>
+                <li>Services</li>
+                <li>Motivational Reading</li>
+                <li>Blog</li>
+                <li>Contact</li>
+            </ul> 
+        </nav>
+        <div class="row col-3">
+            <i class="col-4 fa fa-user-circle-o"></i>
+            <p class="col-8 login">Log in</p>
+        </div>
+    </header>
+    <section class="intro">
+        <div>
+            <p class="ambition">AMBITION IS THE FIRST STEP TOWARDS</p>
+        </div>
+        <div>
+            <p class="succees">SUCCEES</p>
+        </div>
+        <button> Book Now </button>
+    </section>
+    <section class="row">
+        <div class="about md-col-6">
+            <div class="md-col-4"> </div>
+            <div class="md-col-8">
+                <div>
+                    <h2>ABOUT ME</h2>
+                </div>
+                <div>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                       sed do eiusmod tempor incididunt ut labore et dolore
+                       magna aliqua.
+                    </p>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                       sed do eiusmod tempor incididunt ut labore et dolore
+                       magna aliqua.
+                    </p>
+                </div>
+                <div>
+                    <button>Read more</button>
+                </div>
+            </div>
+        </div>
+        <div class="photo col-12 md-col-6"></div>
+    </section>
+    <section class="row pricing">
+        <div class="md-col-2"></div>
+        <div class="md-col-8">
+            <div class="heading">
+                <p>HOW I CAN HELP YOU</p>
```

Comment:
I think it makes more sense to make these `h1`, `h2` or `h3` tags and then style them all together. 

Using the `H` tags makes the HTML easier to read at a glance because you can tell what is meant to be a title and what is meant to be normal text. This is a really small problem though!